features/claims/litigator/scheme_nine/transfer_claim_draft_submit.feature

```diff
@@ -15,7 +15,7 @@ Feature: Litigator partially fills out a draft transfer claim, then later edits
     Then I choose the litigator type option 'New'
     And I choose the elected case option 'No'
     And I select the transfer stage 'Before trial transfer'
-    And I enter the transfer date '2015-05-21'
+    And I enter the transfer date '#{date.year}-05-21'
```

Comment:
It may be better to have a step 'I enter the transfer date (.*) year(s) ago'. It is a bit confusing having `date` defined in the step definition and then used here.

Reply:
Yes - that is a much better idea. :)